src/ui/components/ExplorerHome/ExplorerHome.js

```diff
@@ -91,10 +91,11 @@ const useStyles = makeStyles((theme) => ({
 }));
 
 type ExplorerHomeProps = {|
-  +initialView: CredView,
+  +initialView: CredView | null,
 |};
 
 export const ExplorerHome = ({initialView}: ExplorerHomeProps): ReactNode => {
+  if (!initialView) return null;
```

Comment:
You can also render a Redirect component here in case someone ends up on this page with no cred view

Reply:
This is a backend WIP page, and this line will be changed before the launch since the page will not use a CredView object.